app/views/rails_blogger_engine/articles/show.html.erb

```diff
@@ -9,7 +9,7 @@
     Submitted <%= time_ago_in_words(@article.created_at) %>
     <%= link_to 'Edit', edit_article_path(@article) %> |
     <%= link_to 'Back', articles_path %> |
-    <%= button_to 'Destroy', articles_path(@article), method: :delete, data: { confirm: 'Are you sure?' } %>
+    <%= button_to 'Destroy', @article, method: :delete, data: { confirm: 'Are you sure?' } %>
```

Comment:
This is where the error was happening. I simplified the articles_path(@article) to just @article and now it works.